src/main/java/com/dropbox/core/IncludeGrantedScopes.java

```diff
@@ -0,0 +1,29 @@
+package com.dropbox.core;
+
+/**
+ *  If this field is present, Dropbox server will return a token with all scopes user previously
+ *  granted your app.
+ *  Non-mobile apps use it in {@link DbxWebAuth}.
+ *  Mobile apps use it in {@link com.dropbox.core.android.Auth}
+ */
+public enum IncludeGrantedScopes {
+    /**
+     * Use it in user flow.
+     */
+    USER("user"),
+    /**
+     * Use it in team flow.
+     */
+    TEAM("team");
+
+    private String string;
```

Comment:
If you decide to keep this string, please name it something more descriptive.  Maybe tagName?